segment/functions/track-sms.js

```diff
@@ -0,0 +1,25 @@
+exports.handler = async function(context, event, callback) {
```

Comment:
we can't check headers today in Functions to verify that the webhook comes from Twilio but I wonder if we should add something "unguessable" into the URL via query parameters? @myfancypants ?